src/dotnet/Program.cs

```diff
@@ -113,9 +113,25 @@ private static int ProcessArgs(string[] args)
 
             var appArgs = (lastArg + 1) >= args.Length ? Enumerable.Empty<string>() : args.Skip(lastArg + 1).ToArray();
 
-            if (string.IsNullOrEmpty(command) || command.Equals("help", StringComparison.OrdinalIgnoreCase))
+            if (string.IsNullOrEmpty(command))
```

Comment:
I pushed a new commit that has this all moved to a separate method. Let me know if it looks better.